docs/components/card/examples/CardDefaultExample.vue

```diff
@@ -0,0 +1,12 @@
+<template>
+  <div class="vp-raw inline-flex align-center gap-2 flex-wrap">
+    <Card :href="'#'">
+      <h5 class="mb-2 text-2xl font-bold tracking-tight text-gray-900 dark:text-white">Noteworthy technology acquisitions 2021</h5>
+      <p class="font-normal text-gray-700 dark:text-gray-400">Here are the biggest enterprise technology acquisitions of 2021 so far, in reverse chronological order.</p>
+    </Card>
+  </div>
+</template>
+
+<script setup>
+import { Card } from '../../../../src/index'
```

Comment:
'Component names should always be multi-word' – Vue style guide([link](https://v2.vuejs.org/v2/style-guide/?redirect=true#Multi-word-component-names-essential))
`TheCard` will be good, I think

Reply:
Should I change the file name as well?

Reply:
Yep